### PR TITLE
API Updates

### PR DESCRIPTION
**Pull request by richardm-stripe**
Codegen for openapi 780fbc3.
r? @richardm-stripe
cc @stripe/api-libraries

## Changelog
* Add support for new resource `Quote`
* Add support for `quote` on `Invoice`

### CODE REVIEW

**Comment by richardm-stripe** [comment]
This LGTM now! r? @dcr-stripe

File: src/main/java/com/stripe/param/QuotePdfParams.java [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    public QuotePdfParams build() {
+      return new QuotePdfParams(this.expand, this.extraParams);
+    }
+
+    /**
+     * Add an element to `expand` list. A list is initialized for the first `add/addAll` call, and
+     * subsequent calls adds additional elements to the original list. See {@link
+     * QuotePdfParams#expand} for the field documentation.
+     */
+    public Builder addExpand(String element) {
```
Comment (dcr-stripe) [inline src/main/java/com/stripe/param/QuotePdfParams.java:51]: I wonder if we want these expand properties on the /pdf endpoint? It doesn't really make sense but I suppose the implementation could differ based on the expand request...

File: src/test/java/com/stripe/functional/QuoteTest.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+import com.stripe.model.Quote;
+import com.stripe.model.QuoteCollection;
+import com.stripe.net.ApiResource;
+import com.stripe.util.StreamUtils;
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+public class QuoteTest extends BaseStripeTest {
```
Comment (dcr-stripe) [inline src/test/java/com/stripe/functional/QuoteTest.java:17]: Do we want to add tests for `Computed`?

**Review by dcr-stripe — APPROVED** [review]
Overall LGTM! Thanks Richard!

File: src/main/java/com/stripe/param/QuotePdfParams.java (hunk 1 above)
Comment (richardm-stripe) [reply src/main/java/com/stripe/param/QuotePdfParams.java:51]: The problem originates with the OpenAPI spec which too eagerly defines "expand" params. Out of scope for now.

File: src/test/java/com/stripe/functional/QuoteTest.java (hunk 2 above)
Comment (richardm-stripe) [reply src/test/java/com/stripe/functional/QuoteTest.java:17]: I've done some testing manually. The [fixture](https://github.com/stripe/openapi/blob/master/openapi/fixtures3.yaml#L1846) that we have from stripe-mock doesn't include line_items, so the regression test wouldn't be a very useful one really.

**Pull request merged**